modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7128,7 +7128,7 @@ groups=Groups
 groups-and-roles=Groups and Roles
 groups-dn=Groups DN
 groups-dn-help=New groups will be created at the specified DN. This will only be active if the property "LDAPAuthConfiguration.enabled" and "LDAPExportConfiguration.enabled" are set to true.
-gtin=GTIN
+gtin=GTIN:
```

Comment:
@dsitu is it ok to have **:** character at the end of the line?

Reply:
No, it isn't as far as I know. We can add gtin-x and pass the argument for our use case, I think. Right @dsitu ?

Reply:
@andrea-ale-sbarra  yeah, I believe that this is ok. There are a few like `pin-size = Pin Size:`.
I also saw some named `search-colon=Search:`. They add `-colon` when they also have a language key requirement for `search=Search`.

Reply:
So it will be gtin-colon